sklearn/feature_selection/__init__.py

```diff
@@ -29,10 +32,12 @@
            'SelectFpr',
            'SelectFwe',
            'SelectKBest',
+           'SelectFromModel',
            'SelectPercentile',
            'VarianceThreshold',
            'chi2',
            'f_classif',
            'f_oneway',
            'f_regression',
-           'SelectFromModel']
```

Comment:
Looks like I moved it after `SelectKBest`. I probably wanted all functions to be in the end.